packages/flutter_tools/gradle/flutter.gradle

```diff
@@ -412,14 +396,6 @@ class FlutterPlugin implements Plugin<Project> {
                     dependsOn packageAssets
                     dependsOn cleanPackageAssets
                     into packageAssets.outputDir
-
-                    File chosenFlutterJar = flutterJarFor(flutterBuildMode)
```

Comment:
nit: can the `copyFlutterAssetsTask` be rewritten so that the block under `else` is the main `if` case now? From what I can tell it looks like the `if` block containing this is a non-op now.